environments/staging/staging-hearings-reform-hmcts-net.yml

```diff
@@ -0,0 +1,13 @@
+name: "stagings-hearings-reform.hmcts.net"
```

Comment:
this is not the same zone as in the ticket https://tools.hmcts.net/jira/browse/DTSPO-12579

`.staging.hearings.reform.hmcts.net`

Also you likely need a `hearings.reform.hmcts.net` zone as well before you go to staging.

This is probably something for the service team to takeover as they look after those zones (James Ferrari probably)

Reply:
corrected to .staging.hearings.reform.hmcts.net